.circleci/config.yml

```diff
@@ -193,7 +193,8 @@ jobs:
           name: Run pytest
           command: |
             source $(pyenv root)/versions/3.10.4/envs/env/bin/activate
-            ENVIRONMENT=test pytest --cov-report xml:test-results/coverage.xml --cov=boxtribute_server
+            # overwrite environment variable because it must not depend on the CircleCI context
+            ENVIRONMENT=development MYSQL_PORT=3306 pytest --cov-report xml:test-results/coverage.xml --cov=boxtribute_server
```

Comment:
weird that the socket is needed here if you are using a socket

Reply:
What do you mean? weird that the _port_ is needed? It's because the connection to the DB for integration tests is via TCP. Socket will only be effective in deployed environments.